build-env/target/rhel/8/packages.conf.sh

```diff
@@ -6,7 +6,6 @@ DEPENDS=(
 
 PACKAGES=(
     'pe-nodejs'
-    'devicedb'
```

Comment:
add fluent-bit here?

Reply:
@costanic we currently do not support fluent-bit in rhel 8 - only debian packages.
Once we create a fluent-bit.spec for rhel, we can add it here